README-CONAN.md

```diff
@@ -295,3 +295,12 @@ solution generated by CMake, the dependencies brought by conan will use a specif
 conan dependencies that use a certain Visual Studio Runtime (example: `MD`), and try to compile Exiv2 with other one
 (example: `MT`), you will obtain linking errors. In order to make experiments with different runtime options, it is
 preferable to have different build folders.
+
+### Optional dependencies
+
+We recently added a conna option to add a modern version XMP SDK as a external dependency. This is a experimental feature, and in order to enable the user will need to call conan like this:
```

Comment:
Typo: conna -> conan. 
@clanmills is it "as **a** external dependency" or "as **an** external dependency" (and the same with experimental feature)?

Reply:
It should be ' an experimental ...'  I will fix it too ;)